conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureBodySerDe.java

```diff
@@ -100,7 +100,7 @@ public void serialize(BinaryResponseBody value, HttpServerExchange exchange) thr
     @Override
     public InputStream deserializeInputStream(HttpServerExchange exchange) {
         String contentType = getContentType(exchange);
-        if (!contentType.startsWith(BINARY_CONTENT_TYPE)) {
+        if (!contentType.toLowerCase().startsWith(BINARY_CONTENT_TYPE)) {
```

Comment:
Let's avoid locale-dependent behavior:
```suggestion
        if (!contentType.toLowerCase(Locale.ROOT).startsWith(BINARY_CONTENT_TYPE)) {
```

Reply:
Good flag, updated!